lgn-messages/src/types/v1/preprocessing/db_tasks.rs

```diff
@@ -1,392 +0,0 @@
-use alloy_primitives::U256;
-use derive_debug_plus::Dbg;
-use mp2_common::types::HashOutput;
-use serde_derive::Deserialize;
-use serde_derive::Serialize;
-
-use crate::types::v1::preprocessing::db_keys;
-use crate::types::v1::preprocessing::ext_keys;
-use crate::types::v1::preprocessing::ext_tasks::Identifier;
-use crate::types::v1::preprocessing::WorkerTask;
-use crate::types::v1::preprocessing::WorkerTaskType;
-use crate::BlockNr;
-use crate::TableId;
-
-#[derive(Clone, Dbg, PartialEq, Deserialize, Serialize)]
-pub enum DatabaseType {
-    #[serde(rename = "1")]
-    Cell(DbCellType),
-
-    #[serde(rename = "2")]
-    Row(DbRowType),
-
-    #[serde(rename = "3")]
-    Index(IndexInputs),
-
-    IVC(IvcInput),
-}
```

Comment:
Most of these are wrappers around the mp2 circuit inputs. Now these types implement both serialize and deserialize, the code now uses that instead. This means that adding new messages to the mp2 shouldn't need much changes to the worker from now on. Only optimization work, i.e. batching, needs custom messages.

Because of that most of the messages are dropped.